### DISCUSSION TITLE
The Remote Fleet Management template what screen creates the Analytics

### DISCUSSION THREAD

**Discussion by cngi1** [Budibase/budibase#5426] [Help] [answered]
I am a NOB. I am using the Remote Fleet Management template as a learning guide.  There is a navigation link /analytics that paints charts.  But I don't understand how this page is built since there are no screens in the template that serve this link. Please direct me to the screen that builds this page. 

Thank you.

**Comment by joebudi** [comment, marked as answer]
Hi. Did you try changing the role access (drop-down under screens)?

**Reply by cngi1** [reply]
No I did not until a sec before I came back here to kill the question.  Thanks for the response.
